demisto_sdk/commands/common/update_id_set.py

```diff
@@ -693,6 +693,26 @@ def create_common_entity_data(path, name, to_version, from_version, pack):
     return data
 
 
+def get_pack_metadata_data(path):
+    json_data = get_json(path)
+
+    data = {
+        "name": json_data.get('name', ''),
```

Comment:
could it ever not be there? could this ever return `''`?

Reply:
done for current_version as well